internal/dataplane/parser/translate_kong_l4.go

```diff
@@ -27,17 +26,10 @@ func (p *Parser) ingressRulesFromTCPIngressV1beta1() ingressRules {
 	})
 
 	for _, ingress := range ingressList {
-		if p.featureFlags.ExpressionRoutes && p.kongVersion.LT(versions.ExpressionRouterL4Cutoff) {
```

Comment:
We aren't we checking the expression routes feature flag 🤔 ?

Reply:
It's only for registering error messages and skipping for router & Kong version mismatch, the actual check of router type happens in the line 93

Reply:
👍